tensorflow/lite/allocation.cc

```diff
@@ -32,15 +32,11 @@ namespace tflite {
 FileCopyAllocation::FileCopyAllocation(const char* filename,
                                        ErrorReporter* error_reporter)
     : Allocation(error_reporter) {
-  // Obtain the file size, using an alternative method that is does not
```

Comment:
Thanks for this change! I think better would be to put an accurate comment. Something like:

"Obtain the file size using fstat, or report an error if that fails."